wordle-web/src/views/WordleView.vue

```diff
@@ -1,37 +1,156 @@
+
+  <div class="text-h4 text-center">Wordle Mind Bender</div>
 
   <GameBoard :game="game" @letterClick="addChar" />
 
-  <KeyBoard @letterClick="addChar" :guessedLetters="game.guessedLetters" />
+  <GameKeyboard :guessedLetters="game.guessedLetters" @letterClick="addChar" />
+
+  <v-row class="justify-center">
+    <v-btn
+      @click="checkGuess()"
+      @keyup.enter="checkGuess()"
+      color="primary"
+      size="x-large"
+      v-if="game.status == WordleGameStatus.Active"
+    >
+      Check
+    </v-btn>
+    <v-btn
+      @click="newGame"
+      @keyup.enter="checkGuess"
+      color="secondary"
+      size="x-large"
+      v-if="game.status !== WordleGameStatus.Active"
+    >
+      New Game
+    </v-btn>
+  </v-row>
 
-  <v-btn @click="checkGuess" @keyup.enter="checkGuess"> Check </v-btn>
+  <div class="text-h4 text-center mt-10" v-if="game.status == WordleGameStatus.Lost">
+    Better Luck Next Time
+  </div>
+  <div class="text-h4 text-center mt-10" v-if="game.status == WordleGameStatus.Won">You Won!</div>
 
-  <h2>{{ guess }}</h2>
-  <h3>{{ game.secretWord }}</h3>
+  <v-row class="justify-center" v-if="game.status == WordleGameStatus.Active">
+    <v-col cols="3">
+      <WordleSolver :game="game" @wordClick="(value: string) => checkGuess(value)"></WordleSolver>
+    </v-col>
+  </v-row>
+  <v-row class="justify-center mt-10">
+    <v-btn @click="addWord()" style="tonal" size="x-small">Add Word Test</v-btn>
+  </v-row>
+  <!-- <h2>{{ guess }}</h2> -->
+  <!-- <h3>{{ game.secretWord }}</h3> -->
 </template>
 
 <script setup lang="ts">
-import { WordleGame } from '@/scripts/wordleGame'
+import { WordleGame, WordleGameStatus } from '@/scripts/wordleGame'
 import { ref, reactive, onMounted, onUnmounted } from 'vue'
-import GameBoard from '../components/GameBoard.vue'
-import KeyBoard from '../components/KeyBoard.vue'
 import type { Letter } from '@/scripts/letter'
+import Axios from 'axios'
+import GameBoard from '../components/GameBoard.vue'
+import GameKeyboard from '../components/GameKeyboard.vue'
+import WordleSolver from '../components/WordleSolver.vue'
+import { WordsService } from '@/scripts/wordsService'
 
 const guess = ref('')
 const game = reactive(new WordleGame())
+const namePrompt = ref(false)
+const overlay = ref(true)
+let userName = ref('Guest')
+let attempts = 0
+
+// Start a new game
+newGame()
 
 onMounted(async () => {
   window.addEventListener('keyup', keyPress)
-  await game.restartGame()
-  console.log(game.secretWord)
 })
 onUnmounted(() => {
   window.removeEventListener('keyup', keyPress)
 })
 
-function checkGuess() {
+function promptUser() {
+  if (game.status == WordleGameStatus.Won) {
+    namePrompt.value = true
+  }
+}
+
+function changeUsername() {
+  namePrompt.value = true
+}
+
+function setUsername() {
+  namePrompt.value = false
+  Axios.post('/player', {
+    name: userName.value,
+    attempts: attempts,
+    secondsInGame: 0
+  })
+    .then((response) => {
+      namePrompt.value = false
```

Comment:
you set namePrompt.value to false twice in a row

Reply:
I'll ask my partner about it.

Reply:
alright, removed